clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -1404,8 +1404,22 @@ static Error UnbundleFiles() {
     auto Output = Worklist.find(CurTriple);
     // The file may have more bundles for other targets, that we don't care
     // about. Therefore, move on to the next triple
-    if (Output == Worklist.end())
-      continue;
+    if (Output == Worklist.end()) {
+      // FIXME: temporary solution for supporting binaries produced by old
```

Comment:
Is there any plan for removing this workaround?

Reply:
Yes. The plan is to keep this work-around for 1-2 releases to let our users re-build their software with new compiler.
I wish the bundler to treat triple part of the bundle ID as LLVM triple rather than just a string. We could avoid changing the bundler tool in this case.
Do you have any ideas if we can do better?
At some point I was thinking about diagnostics here, but it's not clear if it will be useful for the users and bundler doesn't seem to have facilities for that.

BTW, I don't have plans to upstream this change to llorg. I'm going to maintain it here until we remove it.

Reply:
Okay, sounds good. I do not have ideas how to handle it better.